benefit/data/sheets/WorkBasketInputFields/WorkBasketInputFields-sandl-nonprod.json

```diff
@@ -1,8 +1,8 @@
 [
   {"LiveFrom": "10/10/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-dwpresponsewriter", "CaseFieldID": "dwpIsOfficerAttending", "Label": "Is the Presenting officer attending", "DisplayOrder": 10},
```

Comment:
Needs bumping

Reply:
anything below 11 should not be needed bump as new one added with order 12

Reply:
conflicts with:
{"LiveFrom": "18/06/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-dwpresponsewriter", "CaseFieldID": "createdInGapsFrom", "Label": "Is a new digital case?", "DisplayOrder": 10},